### PR TITLE
Docs: [AEA-4041] - APIM review of PSU API

### PR DESCRIPTION
**Pull request by JackSpagnoliNHS**
## Summary

- Routine Change

### Details

Makes changes to OAS according to APIM feedback

### CODE REVIEW

File: packages/specification/eps-prescription-status-update-api.yaml [hunk 1]
```diff
@@ -53,15 +55,37 @@ info:
...
-    This API only supports application-restricted access, meaning we authenticate the calling application but not the end user.
+    This API only supports [application-restricted access](https://digital.nhs.uk/developer/guides-and-documentation/security-and-authorisation#application-restricted-apis), meaning we authenticate the calling application but not the end user.
 
+    ### Application-restricted Access
     The end user could be:
-    a healthcare worker - in which case you must ensure they are authenticated and suitably authorised locally
-    not present at all - for example as part of a back end process integrated with a shipping system
+
+    * a healthcare worker - in which case you must ensure they are authenticated and suitably authorised locally
+    * a patient - in which case you must ensure they are authenticated locally
```
Comment (MatthewPopat-NHS) [inline packages/specification/eps-prescription-status-update-api.yaml:64]: It wouldn't be used by a patient, we excluded that intentionally

File: packages/specification/eps-prescription-status-update-api.yaml [hunk 2]
```diff
@@ -53,15 +55,37 @@ info:
...
+    * 400 to 499 if it failed because of a client error by your application
+    * 500 to 599 if it failed because of an error on our server
+
+    Errors specific to each API are shown in the Endpoints section, under Response. See our [reference guide](https://digital.nhs.uk/developer/guides-and-documentation/reference-guide#http-status-codes) for more on errors.
+
+    ## Open source
+    You might find the following [open source](https://digital.nhs.uk/developer/guides-and-documentation/reference-guide#open-source) resources useful:
+
+    | Resource | Description | Links |
+    |----------|-------------|-------|
```
Comment (MatthewPopat-NHS) [inline packages/specification/eps-prescription-status-update-api.yaml:85]: we should have links to this repo and the FHIR validator lambda repo here.

File: packages/specification/eps-prescription-status-update-api.yaml (hunk 1 above)
Comment (JackSpagnoliNHS) [reply packages/specification/eps-prescription-status-update-api.yaml]: 0e9845dc631bf9cd656b409d4616911c2b3f874d

File: packages/specification/eps-prescription-status-update-api.yaml (hunk 2 above)
Comment (JackSpagnoliNHS) [reply packages/specification/eps-prescription-status-update-api.yaml:84]: 5df2893b05761db3c7e45188592ee1fcae43a819

File: packages/specification/eps-prescription-status-update-api.yaml [hunk 3]
```diff
@@ -53,15 +55,36 @@ info:
...
-    This API only supports application-restricted access, meaning we authenticate the calling application but not the end user.
+    This API only supports [application-restricted access](https://digital.nhs.uk/developer/guides-and-documentation/security-and-authorisation#application-restricted-apis), meaning we authenticate the calling application but not the end user.
 
+    ### Application-restricted Access
     The end user could be:
-    a healthcare worker - in which case you must ensure they are authenticated and suitably authorised locally
-    not present at all - for example as part of a back end process integrated with a shipping system
+
+    * a healthcare worker - in which case you must ensure they are authenticated and suitably authorised locally
+    * not present at all - for example as part of a back end process to check NHS numbers for data flowing from one system to another
```
[the remark this answers is not in the document]
Comment (JackSpagnoliNHS) [reply packages/specification/eps-prescription-status-update-api.yaml]: 130993e30f2afd50e1dfce7fdbc9b8a57c3c26c4

**Review by MatthewPopat-NHS — APPROVED** [review]

**Pull request merged**